.github/workflows/tests.yml

```diff
@@ -104,9 +104,14 @@ jobs:
           # brew style doesn't like world writable directories
           sudo chmod -R g-w,o-w "$HOMEBREW_REPOSITORY/Library/Taps"
 
+          echo "HOMEBREW_FORCE_HOMEBREW_ON_LINUX=1" >> $GITHUB_ENV
```

Comment:
Can this be set instead using `env:` through the actions configuration (like other variables)?

Reply:
I suppose it's only needed for the `brew audit` step - it just seemed more "correct" to have it set for all steps after changing the homebrew-core origin.

Reply:
Yeh, just use `env` on those steps (or that step) then I think